src/view/TerminalInterface.py

```diff
@@ -247,7 +247,23 @@ def getConfirmation(self, message, okStr="Y", nokStr="N"):
         okStr = okStr.lower()
         nokStr = nokStr.lower()
         self.__boldPrint(message + f" [{okStr}/{nokStr}]: ")
-        choice = str(self.__getUserInput())
+
+        while True:
+            if okStr == "scratch":
+                choice = str(self.__getUserInput())
+                break
+            elif keyboard.is_pressed("Y"):
+                choice = "y"
+                keyboard.press("backspace")
+                time.sleep(0.2)

```

Comment:
? should this also be in the next elif block?

Reply:
it shouldnt be im sure